### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by dipeshparashar** [smhaley/react-text-annotate-blend#6]
Does not install. If does with --legacy-peer-deps flag > breaks code 
![Screen Shot 2022-07-07 at 00 57 27 AM](https://user-images.githubusercontent.com/8472277/177632214-a07ee6c7-c6fd-4b40-b766-73fd6676c2b9.png)

**Comment by smhaley** [comment]
This is not a bug. This is due to the peer deps requiring React 16. It looks like you are attempting to install with React `17+`. I will test and release a patch to include react 17 and 18 as peer dependencies. 

See package lock: https://github.com/smhaley/react-text-annotate-blend/blob/main/package.json#L43

I have also tested functionality with ` --legacy-peer-dep` on React 18. I am getting full code functionality without error. 

If you need this working asap (before I get out a new release), please share your code so I can recreate the issue.

**Comment by smhaley** [comment]
Following up. I've released a patch on npm: `1.0.5` (https://www.npmjs.com/package/react-text-annotate-blend).

This patch increases the peerDeps to cover React 16, 17, and 18. This should fix your install issues.

I've also included a directory with minimal examples to help get you started: https://github.com/smhaley/react-text-annotate-blend/tree/main/examples

This patch closes out the issues. Example codes has been tested on all supported React versions.
